pkg/stack/stack_processor.go

```diff
@@ -149,88 +160,104 @@ func ProcessYAMLConfigFile(
 	relativeFilePath := u.TrimBasePathFromPath(basePath+"/", filePath)
 
 	stackYamlConfig, err := getFileContent(filePath)
-	if err != nil {
+	if err != nil && !ignoreMissingFiles {
```

Comment:
Nit: Could we comment this line. It's confusing to read because it's basically saying if errs is not empty and we're not ignoring missing files, do an early return. 

It would be nice to state something like, if error and were acknowledging missing files, early return.